module/apmazure/storage.go

```diff
@@ -127,6 +131,7 @@ type azureRPC interface {
 	name() string
 	_type() string
 	subtype() string
+	serviceName() string
```

Comment:
```suggestion
	targetName() string
```

Otherwise it sounds a bit like it would return the cloud service name, like "azure service bus".

Reply:
Good point! Changed :+1: